src/dynamodb.js

```diff
@@ -154,11 +128,11 @@ exports.tableSchema = (schema) => {
   tablesSchema = cloneDeep(schema);
 };
 
-function dynamoDBTestHooks (useUniqueTables = false) {
+function dynamoDBTestHooks (useUniqueTables = false, externalConfig) {
```

Comment:
This seems like it unnecessarily exposes library internals. If we have a real client configuration issue then I would expect that we could simply adjust the defaults internally.

Reply:
I am having real issues with the current defaults.  I get so many connection resets or timeouts from the 3 second limit, and only 3 retries that I can't run LifeService tests locally.

Reply:
Sure. So can't we just set more lenient default here? Do we really need to make the configuration customizable?

Reply:
I have done that now, and removed the ability to customize the connection from outside